udata_front/theme/gouvfr/templates/dataservice/display.html

```diff
@@ -99,7 +99,11 @@ <h2 id="description" class="subtitle fr-mb-1w">{{ _('Description') }}</h2>
         </section>
         <section class="fr-col-12 fr-col-md-4">
             {% if dataservice.organization %}
-                <h2 id="producer" class="subtitle fr-mb-1v">{{ _('Producer') }}</h2>
+                {% if dataservice.contact_points|selectattr("role", "publiser") or dataservice.contact_points|selectattr("role", "creator") %}
+                    <h2 id="organization" class="subtitle fr-mb-1v">{{ _('Organization') }}</h2>
+                {% else %}
+                    <h2 id="producer" class="subtitle fr-mb-1v">{{ _('Producer') }}</h2>
+                {% endif %}
```

Comment:
We don't want to dive into Organization vs Producer yet I think.
It should probably done when we add all contact roles